src/server/set_family.cc

```diff
@@ -284,19 +284,32 @@ void InterStrSet(const DbContext& db_context, const vector<SetType>& vec, String
   }
 }
 
-StringVec PopStrSet(const DbContext& db_context, unsigned count, const SetType& st) {
+StringVec RandMemberStrSet(const DbContext& db_context, const SetType& st,
+                           const std::unique_ptr<PicksGenerator>& generator,
```

Comment:
Would a virtual function making it even slower?

Reply:
1. You can pass just const PicksGenerator&
2. I don't think the virtual function call adds any mesurable latency 🙂

Reply:
I agree with @dranikpg, keep the virtual function call as it was